src/ui/wxWidgets/I18N/pos/pwsafe_de.po

```diff
@@ -2439,179 +2419,188 @@ msgid "Show Menu Separator"
 msgstr "Menütrenner anzeigen"
 
 #: ../../../ui/wxWidgets/OptionsPropertySheetDlg.cpp:538
+msgid "Show Optimized Cell size"
+msgstr "Zeige optimierte Zell-Größe im Grid"
```

Comment:
Since "Grid" refers to the "List View", I would suggest using a similar German term for "Grid" as it is already used in the application.
Additionally, based on the already available translations, I would also suggest putting the verb at the end of the sentence.

For instance:
> Optimierte Zellgröße in der Tabellenansicht anzeigen

Reply:
I have overtaken the proposal, thanks a lot.